Hymns/Tags/TagStore.swift

```diff
@@ -9,6 +9,7 @@ protocol TagStore {
     func observeTagStatus(hymnIdentifier: HymnIdentifier, action: @escaping (Bool) -> Void) -> Notification
     func querySelectedTags(tagSelected: String?) -> Results<TagEntity>
     func queryTagsForHymn(hymnIdentifier: HymnIdentifier) -> Results<TagEntity>
+    func queryUniqueTags() -> [TagEntity]
```

Comment:
If you're just getting the tags, I'd say this should probably be a `[String]` not a `[TagEntity]`, since you don't really care about any of the other stuff

Reply:
After changing the function it now returns a Results<TagEntity> like all the other get/query functions. The problem with making it a string is then it won't fit into the same variable @Published var tags  at the viewModel level. Should I still make it a [String]

Reply:
I don't understand what you mean by "won't fit into the same variable". Can't you just change the view model be Result<String> instead of Result<TagEntity>?

Reply:
Do you still want me to try to do something here or are we good to merge this PR?

Reply:
oh, yeah. Could you try to get it to return [String] instead of [TagEntity]?